src/reference/asciidoc/http.adoc

```diff
@@ -58,6 +58,26 @@ If it does in fact locate that bean, then the support for MultipartFiles will be
 Otherwise, it will fail when trying to map a multipart-file request to a Spring Integration Message.
 For more on Spring's support for MultipartResolvers, refer to the http://static.springsource.org/spring/docs/current/spring-framework-reference/html/mvc.html#mvc-multipart[Spring Reference Manual].
 
+[NOTE]
+====
+If you wish to proxy a `multipart/form-data` to another server, it may be better to keep it in raw form.
+To configure for this situation, do not configure the `multipartResolver` bean, configure the endpoint to expect
```

Comment:
What is bad here that we may want to use `multipartResolver` for one service, but don't want for another and just proxy like we have here.

I mean that this `multipartResolver` global bean logic isn't so flexible.
But yes, it's a part of Spring MVC already...

CC @rstoyanchev